tests/test-buttercup.el

```diff
@@ -613,27 +613,40 @@
 ;;; Spies
 
 (describe "The Spy "
-  (let (saved-test-function saved-test-command)
+  (let (saved-test-function
+        saved-test-command
+        saved-test-function-throws-on-negative)
     ;; We use `before-all' here because some tests need to access the
     ;; same function as previous tests in order to work, so overriding
     ;; the function before each test would invalidate those tests.
+    ;; Unfortunately there's no way to do this lexically, so these
+    ;; function definitions are leaked after the tests run.
```

Comment:
Do they? Isn't `fmakunbound` in the after-each enough to remove them?

Reply:
Yes, that comment is left over from an earlier version of the tests that did leak the function definitions. I'll delete it